cs/style.css

```diff
@@ -0,0 +1,26 @@
+body {
+	background-color: yellow;
+}
+
+#container {
+	text-align: center;
+	margin-left: auto;
+	margin-right: auto;
+}
+
+.centered {
```

Comment:
You almost had horizontal centering working. text-align: center should be on the parent element, and margin-left and margin-right: auto should be on the element to be centered, within the parent element.